### PR TITLE
update CHANGELOG.md

### PR DESCRIPTION
**Pull request by vieux**

### CODE REVIEW

**Comment by vieux** [comment]
ping @nishanttotla @dongluochen @abronan

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,6 +50,7 @@
 
 #### Misc
 
+- Update golang to 1.5.4
```
Comment (nishanttotla) [inline CHANGELOG.md]: nit: `golang version`? 😸

**Comment by nishanttotla** [comment]
LGTM

**Comment by abronan** [comment]
LGTM

**Comment by dongluochen** [comment]
LGTM

**Pull request merged**